base/api/generated/models/DbiamPersonenkontexteUpdateError.ts

```diff
@@ -46,7 +46,8 @@ export const DbiamPersonenkontexteUpdateErrorI18nKeyEnum = {
     PersonIdMismatch: 'PERSON_ID_MISMATCH',
     PersonNotFound: 'PERSON_NOT_FOUND',
     InvalidPersonenkontextForPersonWithRollenartLern: 'INVALID_PERSONENKONTEXT_FOR_PERSON_WITH_ROLLENART_LERN',
-    BefristungRequiredForPersonenkontext: ' BEFRISTUNG_REQUIRED_FOR_PERSONENKONTEXT'
+    BefristungRequiredForPersonenkontext: ' BEFRISTUNG_REQUIRED_FOR_PERSONENKONTEXT',
```

Comment:
just for awareness: this looks like a typo in the API. since we shouldn't touch these generated files, maybe we'll have to fix this in the backend, if the space causes trouble
```suggestion
    BefristungRequiredForPersonenkontext: 'BEFRISTUNG_REQUIRED_FOR_PERSONENKONTEXT',
```